### PR TITLE
Update alumni information

### LINKED ISSUE #439: Alumni info is out of date
Details on the alumni page is now out of date as former team members have changed roles over time (MC, TA, others?). 

It might be easier to just remove any mention of their new / current roles to prevent this from occuring?

### PR DESCRIPTION
**Pull request by ubdbra001**
Adjusted to show where the alumni moved onto immediately after leaving the RSE team, and when they left. Closes #439.

I think I've got most of the info correct, but it'd be useful if someone who has been around for a bit longer than me could double check.

Thanks!

### CODE REVIEW

File: _people/john-charlton.md [hunk 1]
```diff
@@ -10,7 +10,8 @@ role: Research Software Engineer
 ---
 
 John was a PhD student and Research Software Engineer. Previous work had involved simulating dense crowds of virtual pedestrians, using GPUs to model many tens of thousands of people in real time.
-His interests included agent-based modelling, visualisation and interaction of simulations. Expertise includes GPU-accelerated computing and agent-based modelling approaches.
+His interests included agent-based modelling, visualisation and interaction of simulations. Expertise includes GPU-accelerated computing and agent-based modelling approaches.  
+John left the RSE Sheffield team in 2020 to take up a role as a Postdoctoral Research Fellow at the University of Sheffield, in Neopath, part of the School of Clinical Dentitry
```
Comment (ptheywood) [inline _people/john-charlton.md:14]: Looks like John only started in Neopath/dentistry in Nov 2023. 

https://coolmule0.github.io/about/#neopath

I'm not sure on the best way to describe John's secondment in the team ending, be it to complete his PhD or do other RA work in the uni immediately after.

@coolmule0 - any suggestions / preference?

**Review by ptheywood — CHANGES_REQUESTED** [review]
Mostly looks correct to me, just a question about John's immediate role following the end of his secondment into the team. 

I'm not sure on the actual answer, so have tagged John in the comment.

File: _people/phil-tooley.md [hunk 2]
```diff
@@ -16,7 +16,8 @@ Fortran, but also champions the use of the Numpy/Scipy stack for scientific comp
 This includes the use of accelerator technologies including [Numba](https://numba.pydata.org) and
 [Cython](http://cython.org/) to write custom python code which is speed competitive with
 traditional compiled languages, possible in conjunction with parallel frameworks such as
-[Dask](https://dask.pydata.org/).
+[Dask](https://dask.pydata.org/).  
+Phil left the RSE Sheffield team in 2019 to take up a role as a Senior Software Engineer at the Numerical Algorithms Group (NAG).
 
 * Email: phil.tooley (at) sheffield.ac.uk
```
Comment (ns-rse) [inline _people/phil-tooley.md]: Should we remove this if its not a valid email address?

**Review by ns-rse — APPROVED** [review]
For those I know anything about (Anna, David Wilby, Will, Bob, Fariba, Paul) all looks accurate and reworking of others looks fine, left a comment as to whether Phil Tooley's email address `@sheffield.ac.uk` is still valid. Brief search only returns hits to our own pages!

File: _people/phil-tooley.md (hunk 2 above)
Comment (ubdbra001) [reply _people/phil-tooley.md]: Probably a good shout, same with Bob's email?

Comment (ns-rse) [reply _people/phil-tooley.md]: :facepalm: Yes, Bob's too then, I did scan through to see if there were others but obviously missed that so good :eye: 's

File: _people/john-charlton.md (hunk 1 above)
Comment (coolmule0) [reply _people/john-charlton.md:14]: These suggest changes sound the easiest to read, even if not strictly true  :+1:

Comment (ubdbra001) [reply _people/john-charlton.md:14]: Thanks @coolmule0 
Happy with this @ptheywood ?

**Review by ptheywood — APPROVED** [review]

File: _people/paul-richmond.md [hunk 3]
```diff
@@ -9,7 +9,7 @@ role: Former Group Director
...
+Paul was a co-founder of the RSE team and led its growth and development from three staff to a team of roughly 12 full time employees over a period of six years. He left the RSE Sheffield team in 2023 to become the RSE team lead at the [Cambridge Institute of Computing for Climate Science](https://cambridge-iccs.github.io/), but holds a part time role at the University of Sheffield as a Professor of Research Software Engineering in the Department of Computer Science. He no longer oversees the RSE team but is actively engaged in research projects and impact work relating to the application of research software, in particular the [FLAME GPU](http://www.flamegpu.com) software for GPU accelerated agent based simulation. Paul was formally an [EPSRC Research Software Engineering Fellow](https://rse.ac.uk/community/epsrc-rse-fellows/) and President of the society of Research Software Engineering](https://society-rse.org/about/governance/rse-society-trustees-2020-2021/). He is actively engaged in the national RSE community and retains interest in advancing the national provision of RSE collaboration including within the University of Sheffield.
```
Comment (Robadob) [inline _people/paul-richmond.md:12]: "three" and "12" in the same sentence, consistency pls, albeit not your fault.

**Review by Robadob — APPROVED** [review]

**Pull request merged**